src/Agent.Sdk/Agent.Sdk.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Security.Utilities.Core" Version="1.17.0" />
+    <PackageReference Include="Microsoft.Security.Utilities.Core" Version="[1.17.0-g4bafbeaa94]" />
```

Comment:
Before this can be taken out of draft, this change will require a new 1.18 with the detection callback used in this PR.